bin/completion-handlers.ts

```diff
@@ -14,60 +50,408 @@ export function setupCompletionForPackageManager(
   } else if (packageManager === 'bun') {
     setupBunCompletions(completion);
   }
-
-  // Note: Package manager logic is now handled by PackageManagerCompletion wrapper
 }
 
 export function setupPnpmCompletions(completion: PackageManagerCompletion) {
-  completion.command('add', 'Install a package');
-  completion.command('remove', 'Remove a package');
-  completion.command('install', 'Install dependencies');
-  completion.command('update', 'Update dependencies');
-  completion.command('run', 'Run a script');
-  completion.command('exec', 'Execute a command');
-  completion.command('dlx', 'Run a package without installing');
-  completion.command('create', 'Create a new project');
-  completion.command('init', 'Initialize a new project');
-  completion.command('publish', 'Publish the package');
-  completion.command('pack', 'Create a tarball');
-  completion.command('link', 'Link a package');
-  completion.command('unlink', 'Unlink a package');
-  completion.command('outdated', 'Check for outdated packages');
-  completion.command('audit', 'Run security audit');
-  completion.command('list', 'List installed packages');
+  // Package management
+  const addCmd = completion.command('add', 'Install packages');
```

Comment:
as i mentioned, these options and commands them themselves have values which we need autocompletions for. only adding names won't show the library as a more feasible option compared to the traditional autocompletion scripts that these package managers provide.

Reply:
again, descriptions should be retrieved from --help. 

because these descriptions are volatile and we should always show what the actual package manager shows. otherwise, it'd be an inconsistency between actual descriptions and what we provide.